README.md

```diff
@@ -78,22 +78,11 @@ class MyEmitter
 end
 ```
 
-#### Multiple listeners
-
-It is possible to register more than one listener to a given event by calling [previous_def](https://crystal-lang.org/docs/syntax_and_semantics/methods_and_instance_variables.html) inside the block making the blocks stackable. If the listeners are async, each one is executed in its own fiber concurrently.
-
-```crystal
-on :message do |message|
-  previous_def(message)
-  log "Message received: #{message}"
-end
-```
-
 ### Instance
 
-The class ```EventEmitter::Base``` provides the methods ```on```, ```once``` and ```emit``` to insert a new listener, insert a one-time listener and trigger an event, respectively.
+The class `EventEmitter::Base` provides the methods `on`, `once` and `emit` to insert a new listener, insert a one-time listener and trigger an event, respectively. It also provides a `all` method for responding to all events, and a `remove_listener` method to delete a listener.
```

Comment:
English is not may first language so correct me if I'm wrong, but shouldn't it be like this?

```suggestion
The class `EventEmitter::Base` provides the methods `on`, `once` and `emit` to insert a new listener, insert a one-time listener and trigger an event, respectively. It also provides an `all` method for responding to all events, and a `remove_listener` method to delete a listener.
```

Reply:
You're correct, I missed that